src/target/imxrt.c

```diff
@@ -59,21 +58,57 @@
 #define IMXRT_MPU_BASE UINT32_C(0xe000ed90)
 #define IMXRT_MPU_CTRL (IMXRT_MPU_BASE + 0x04U)
 
-#define IMXRT_CCM_ANALOG_BASE     UINT32_C(0x400d8000)
-#define IMXRT_CCM_ANALOG_PLL3_PFD (IMXRT_CCM_ANALOG_BASE + 0x0f0U)
+#define IMXRT_10XX_CCM_ANALOG_BASE     UINT32_C(0x400d8000)
```

Comment:
Per the naming schemes used in other files in BMD, this should probably be `IMXRT10xx_` not `IMXRT_10XX_` (ditto for each of the defines below)